### ISSUE TITLE
Install Generator doesn't generate in SCSS file

### ISSUE THREAD

**Issue by RichardGregoryHamilton** [rweng/jquery-datatables-rails#154]
I read the documentation, and got as far as step 3.

Running the command

    rails generate jquery:datatables:install

It added `//= require dataTables/jquery.dataTables` to my `application.js` file.

When I checked my `application.css.scss` file, it wasn't there.

Needless to say, datatables did not work.

**Comment by lichtamberg** [comment]
also doesnt work for me.. 

```ruby
rails generate jquery:datatables:install bootstrap3
=> Could not find generator jquery:datatables:install.
```

**Comment by sivagollapalli** [comment]
@lichtamberg I also faced this issue. Please update your Gemfile like as `gem 'jquery-datatables-rails', '~> 3.2'` would you solve your issue. 

@ricardodovalle If css minifest extension is `.css.scss` then install generator wont edit css minifest file. I think it seems an issue. Thoughts please?

**Comment by onebree** [comment]
@sivagollapalli I agree that the generator does not recognize `.css.scss` filetypes. I tried the generate the LOC myself and found that only the JavaScript line was inserted.
